lib/world_cup.rb

```diff
@@ -0,0 +1,76 @@
+require "pry"
+require "./lib/team"
+
+class WorldCup
+  attr_reader :year, :teams
+  def initialize(year, array_of_teams)
+    @year = year
+    @teams = array_of_teams
+  end
+
+  def active_teams
+    @teams.select do |team|
+      team.eliminated == false
+    end
+  end
+
+  def active_players_by_position(position_on_field)
+    return_array = []
+    active_teams.select do |team|
```

Comment:
You aren't actually using the functionality for select. You are basically using it as if it were each. I think this is a big area of improvement for you. You really really really need to know enumerables by the end of mod 1 (really). You could use a map here to map the active teams to there list of players.